### ISSUE TITLE
Improved tooltips render in wrong position when TOC is scrolled

### ISSUE THREAD

**Issue by cbjoldham** [squidfunk/mkdocs-material#3938]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

If you enable the Improved Tooltips using
```
theme:
  features:
    - content.tooltips
```
It interacts poorly with TOC entries using an ellipsis.

### Expected behaviour

Tooltips appear with the full navigation entry when the pointer hovers over an entry shortened with an ellipsis

### Actual behaviour

Tooltips appear correctly until the TOC is vertically scrolled.
After the TOC is vertically scrolled tooltips seem to appear at the _original_ location of the entry on screen.
With sufficient vertically scrolling the tooltips do not appear at all, probably as they are rendered off the viewable area.

### Steps to reproduce

1. Enable the Improved Tooltips experimental feature
2. Have a table of contents of sufficient width to trigger ellipsis and sufficient length to trigger vertical scrolling
3. Observe location of tooltips before and after a small amount of vertical scrolling

### Package versions

Docker Image built from [8.2.15-insiders-4.15.1](https://github.com/squidfunk/mkdocs-material-insiders/releases/tag/8.2.15-insiders-4.15.1)

### Configuration

```yaml
theme:
  features:
    - content.tooltips
```

### System information

Edition	Windows 10 Enterprise
Version	21H2
Installed on	‎10/‎01/‎2022
OS build	19044.1706
Experience	Windows Feature Experience Pack 120.2212.4170.0

Firefox 100.0.1 (64-bit)

**Comment by squidfunk** [comment]
Fixed in [`0097d9d`](https://github.com/squidfunk/mkdocs-material-insiders/commit/0097d9dafc41528cc7caea0bdbce98edf0826916). Tooltips will now honor the scroll progress of the containing element, same as annotations, which in that case is the sidebar:

<img width="311" alt="Bildschirmfoto 2022-05-22 um 14 22 40" src="https://user-images.githubusercontent.com/932156/169694909-fa26a5a1-4e37-4008-ac51-b060acf6645c.png">

**Comment by squidfunk** [comment]
Released as part of 8.2.15+insiders-4.15.2!
